### PR TITLE
mariadb@10.9 10.9.8

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  CMake Error at cmake/libfmt.cmake:45 (MESSAGE):
    system libfmt library is not found or unusable
  Call Stack (most recent call first):
    CMakeLists.txt:395 (CHECK_LIBFMT)
```

**Comment by ZhongRuoyu** [comment]
To be safe, I suggest waiting for upstream maintainers' feedback on the patch first.

**Comment by chenrui333** [comment]
Nice work!

- relates to https://github.com/MariaDB/server/pull/2732

File: Formula/m/mariadb@10.9.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class MariadbAT109 < Formula
   desc "Drop-in replacement for MySQL"
   homepage "https://mariadb.org/"
-  url "https://downloads.mariadb.com/MariaDB/mariadb-10.9.7/source/mariadb-10.9.7.tar.gz"
-  sha256 "59c4b178eb859ead207d5c0a7515263c734dc0622af46e1657ef12cb81bc10f5"
+  url "https://downloads.mariadb.com/MariaDB/mariadb-10.9.8/source/mariadb-10.9.8.tar.gz"
```
[the remark this answers is not in the document]
Comment (ZhongRuoyu) [reply Formula/m/mariadb@10.9.rb]: Applied suggestion.

File: Formula/m/mariadb@10.9.rb [hunk 2]
```diff
@@ -1,10 +1,9 @@
 class MariadbAT109 < Formula
   desc "Drop-in replacement for MySQL"
   homepage "https://mariadb.org/"
-  url "https://downloads.mariadb.com/MariaDB/mariadb-10.9.7/source/mariadb-10.9.7.tar.gz"
-  sha256 "59c4b178eb859ead207d5c0a7515263c734dc0622af46e1657ef12cb81bc10f5"
-  license "GPL-2.0-only"
-  revision 1
+  url "https://archive.mariadb.org/mariadb-10.9.8/source/mariadb-10.9.8.tar.gz"
+  sha256 "fcdc6f73882477c20310e795a02f746e54934aa7a8b368900bd45e78a5daf6ed"
+  license "GPL-2.0-or-later"
```
Comment (ZhongRuoyu) [inline Formula/m/mariadb@10.9.rb:6]: Corrected the license; see https://github.com/MariaDB/server/blob/mariadb-10.9.8/COPYING#L299

**Review by iMichka — APPROVED** [review]

**Pull request merged**